modules/gapi/src/backends/fluid/gfluidcore_func.simd.hpp

```diff
@@ -184,6 +184,14 @@ ABSDIFFC_SIMD(float)
 
 #undef ABSDIFFC_SIMD
 
+#define SPLIT3_SIMD(DST1, DST2, DST3, SRC)                          \
+int split3_simd(const SRC in[], DST1 out1[], DST2 out2[],           \
```

Comment:
It makes no sense to make this function a macro for this kernel. Make this function a normal function.
And There is only DST type. Three types are redundant